packages/oasis-actions/src/operations/aave/borrow.ts

```diff
@@ -3,7 +3,7 @@ import BigNumber from 'bignumber.js'
 import * as actions from '../../actions'
 import { ADDRESSES } from '../../helpers/addresses'
 import { OPERATION_NAMES } from '../../helpers/constants'
-import { Address } from '../../strategies/types'
+import { Address } from '../../strategies/types/StrategyParams'
```

Comment:
is having longer import necessary?

Reply:
I think having one place to import types is for better developer experience

Reply:
No it is not. This is changed